.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: node_js
+node_js:
+  - "6"
+sudo: required
+dist: trusty
+# Cache yarn dependencies
+cache:
+  yarn: true
+  directories:
+    - node_modules
+    # TODO - Review why bower_components are not being cached (see TravisCI log for more info)
+    - bower_components
+before_install:
+  # Before starting the build go to the subfolder where code was placed
+  - cd src
+  # Setup browser
+  - export CHROME_BIN=/usr/bin/google-chrome
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - sudo apt-get update
+  - sudo apt-get install -y libappindicator1 fonts-liberation
+  - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+  - sudo dpkg -i google-chrome*.deb
+before_script:
+  # Install dependencies
+  - yarn
+  # Update the browser webdrivers
+  - node ./node_modules/protractor/bin/webdriver-manager update
```

Comment:
Did you verify if it is not done by `yarn` when executes _postinstall_? see https://github.com/DopplerRelay/relay-webapp/blob/develop/src/package.json#L50